src/lib/components/EmptyResults/EmptyResults.js

```diff
@@ -51,9 +48,34 @@ EmptyResults.propTypes = {
   error: PropTypes.object.isRequired,
   queryString: PropTypes.string,
   resetQuery: PropTypes.func.isRequired,
+  overridableUID: PropTypes.string,
 };
 
 EmptyResults.defaultProps = {
   queryString: '',
-  renderElement: null,
+  overridableUID: '',
+};
+
+const Element = ({ overridableUID, ...props }) => {
+  const { queryString, resetQuery } = props;
+  return (
+    <Overridable
+      id={buildUID('EmptyResults.element', overridableUID)}
+      {...props}
+    >
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="search" />
+          No results found!
+        </Header>
+        <em>Current search "{queryString}"</em>
```

Comment:
There is a small bug in RSK, you can have 0 results but the queryString is empty. Could you please add:
```suggestion
        {queryString && <em>Current search "{queryString}"</em>}
```